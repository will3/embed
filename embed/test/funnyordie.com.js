const test = require('./utils/test');

test('funnyordie.com',
  'http://www.funnyordie.com/videos/e52bef28f8/welcome-to-america-with-gad-elmaleh-and-ron-livingston');
