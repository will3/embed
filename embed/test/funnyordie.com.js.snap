// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` funnyordie.com 1`] = `
Object {
  "description": "Gad Elmaleh Struggles with a U.S. Customs and Border Patrol officer (Ron Livingston), giving the Moroccan born comedian his first taste of The United States Of America.",
  "originalUrl": "http://www.funnyordie.com/videos/e52bef28f8/welcome-to-america-with-gad-elmaleh-and-ron-livingston",
  "site_name": "Funny Or Die",
  "thumbnail": Object {
    "url": "http://p.fod4.com/p/media/e52bef28f8/rmnUDIy6Qbm7W40FPn0L_GAD-Thumb.jpg",
  },
  "title": "Welcome To America with Gad Elmaleh and Ron Livingston from Funny Or Die, Ron Livingsto...",
  "video": Object {
    "description": undefined,
    "height": "400",
    "title": undefined,
    "url": "//www.funnyordie.com/embed/e52bef28f8",
    "width": "640",
  },
}
`;
